### PR TITLE
[Inference] fix openai sdk steaming response

### PR DESCRIPTION
**Pull request by KepingYan**

### CODE REVIEW

**Comment by KepingYan** [comment]
Gently ping @xuechendi, you can retest openai sdk streaming reponse and langchain after this ps is merged.

**Comment by xuechendi** [comment]
@KepingYan , seems "\n\n" will not work, I have done some test in my setup, will ping you offline

File: examples/inference/api_server_openai/query_openai_sdk.py [hunk 1]
```diff
@@ -58,4 +58,8 @@
     temperature=args.temperature,
     top_p=args.top_p,
 )
-print(chat_completion)
+if args.streaming_response:
+    for chunk in chat_completion:
+        print(chunk)
```
Comment (carsonwang) [inline examples/inference/api_server_openai/query_openai_sdk.py:63]: Can we use the openAI example that previous didn't work?
```
for chunk in stream:
    if chunk.choices[0].delta.content is not None:
        print(chunk.choices[0].delta.content, end="")
```

**Review by carsonwang — COMMENTED** [review]
Thank you @KepingYan for the quick fix! Thanks @xuechendi for reporting and investigating this together. We should add a test to ensure this don't break in future. But let's leave it for now and it will be added later.

**Review by carsonwang — APPROVED** [review]
LGTM.

**Pull request merged**